[2025-02-18T21:01:32.984+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T21:01:33.026+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dummy_dag.task4 manual__2025-02-18T21:01:22.328144+00:00 [queued]>
[2025-02-18T21:01:33.048+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dummy_dag.task4 manual__2025-02-18T21:01:22.328144+00:00 [queued]>
[2025-02-18T21:01:33.048+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T21:01:33.084+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task4> on 2025-02-18 21:01:22.328144+00:00
[2025-02-18T21:01:33.096+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=216) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-18T21:01:33.116+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dummy_dag', 'task4', 'manual__2025-02-18T21:01:22.328144+00:00', '--job-id', '1011', '--raw', '--subdir', 'DAGS_FOLDER/15_dummy_dag.py', '--cfg-path', '/tmp/tmp5roxwh_4']
[2025-02-18T21:01:33.118+0000] {standard_task_runner.py:105} INFO - Job 1011: Subtask task4
[2025-02-18T21:01:33.099+0000] {standard_task_runner.py:72} INFO - Started process 219 to run task
[2025-02-18T21:01:33.222+0000] {task_command.py:467} INFO - Running <TaskInstance: dummy_dag.task4 manual__2025-02-18T21:01:22.328144+00:00 [running]> on host ab74d4d90d32
[2025-02-18T21:01:33.366+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dummy_dag' AIRFLOW_CTX_TASK_ID='task4' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T21:01:22.328144+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-18T21:01:22.328144+00:00'
[2025-02-18T21:01:33.368+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T21:01:33.392+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-18T21:01:33.394+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-18T21:01:33.403+0000] {subprocess.py:99} INFO - Output:
[2025-02-18T21:01:38.408+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-18T21:01:38.464+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T21:01:38.465+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dummy_dag, task_id=task4, run_id=manual__2025-02-18T21:01:22.328144+00:00, execution_date=20250218T210122, start_date=20250218T210133, end_date=20250218T210138
[2025-02-18T21:01:38.534+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T21:01:38.565+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T21:01:38.568+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
